src/epoch/ut1.rs

```diff
@@ -82,6 +100,40 @@ pub struct Ut1Provider {
 }
 
 impl Ut1Provider {
+    /// Borrowing iterator over UT1 records (double-ended).
+    ///
+    /// Arguments
+    /// -----------------
+    /// * None.
+    ///
+    /// Return
+    /// ----------
+    /// * A `std::slice::Iter<'_, DeltaTaiUt1>` that supports `.rev()`.
+    ///
+    /// See also
+    /// ------------
+    /// * [`as_slice`] – Direct slice access to underlying data.
+    pub fn iter(&self) -> std::slice::Iter<'_, DeltaTaiUt1> {
```

Comment:
Since `Ut1Provider` now implements the IntoIterator trait, is the `iter` method still needed?

Reply:
done in bfa04de